crates/sui-types/src/messages.rs

```diff
@@ -115,6 +129,8 @@ pub enum SingleTransactionKind {
     Call(MoveCall),
     /// Initiate a SUI coin transfer between addresses
     TransferSui(TransferSui),
```

Comment:
`TransferSui` is just `Pay` with a single recipient and amount, we should consider removing `TransferSui` after we added `Pay` so that people won't be confused about which one to use.

Reply:
The subtle difference between the two is that:
- `Pay` works for any coin type
- `TransferSui` can debit the gas coin, whereas `Pay` cannot (because `Pay` might be operating over a different coin type)